hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/HtsSystemContract.java

```diff
@@ -83,6 +83,7 @@ public FullResult computeFully(@NonNull final Bytes input, @NonNull final Messag
         return resultOfExecuting(attempt, call, input, frame);
     }
 
+    @SuppressWarnings({"java:S2637", "java:S2259"}) // this function is going to be refactored soon.
```

Comment:
Is there a ticket out there to refactor it? You should include the ticket number and a @todo tag

Reply:
@tinker-michaelj told me he is going to do a major refactoring during the last 3 weeks of Decmeber for smart contracts he is aware about it. That's why I didn't raise a ticket.